carma_guidance_plugins/src/control_plugin.cpp

```diff
@@ -74,10 +74,15 @@ namespace carma_guidance_plugins
         [this]() {
           if (this->get_activation_status()) // Only trigger when activated
           {
-            this->vehicle_cmd_pub_->publish(this->generate_command());
+            auto vehicle_cmd = this->generate_command();
+            //If vehicle cmd is outdated - dont publish
+            if ((this->now() - rclcpp::Time(vehicle_cmd.header.stamp)).seconds() < 5.0){
```

Comment:
Separately, I'm trying to think through issues that could arise with this approach. If we start a normal single vehicle test, CARMA Platform will use Pure Pursuit as the controller, but if we then begin platooning with another vehicle, then the system will switch to Platooning Control as the controller. Does this mean that for five seconds (or the value that we have configured here), we'll have overlap where both control nodes will publish commands?

Reply:
After taking another look, in this example I think the overlap between both Pure Pursuit and Platooning Control both generating and publishing valid control commands would happen indefinitely since both control nodes would be able to generate commands in line with the current timestamp. 

Referencing [PurePursuitWrapperNode::generate_command()](https://github.com/usdot-fhwa-stol/carma-platform/blob/a92c44e692e7f36d0ace654dfc0c793769693ecd/pure_pursuit_wrapper/src/pure_pursuit_wrapper.cpp#L132), Pure Pursuit would just continue using the latest received Pure Pursuit trajectory (could be even from a minute ago) and would generate a command for it.

Reply:
The trajectory executor will publish the trajectory to the control plugin that is supposed to generate the control commands https://github.com/usdot-fhwa-stol/carma-platform/blob/a92c44e692e7f36d0ace654dfc0c793769693ecd/trajectory_executor/src/trajectory_executor_node.cpp#L133
So there should be no overlap between any control plugin generating valid commands

Reply:
Resolving this comment since we are removing and updates related to the vehicle command timeout from this PR.